Gems/SceneProcessing/Code/Tests/SceneBuilder/SceneBuilderConfigTests.cpp

```diff
@@ -159,3 +191,145 @@ TEST_F(SceneProcessingConfigTest, SceneProcessingConfigSystemComponent_ScriptCon
+
+    AZ::SceneProcessingConfig::SceneProcessingConfigSystemComponent sceneProcessingConfigSystemComponent;
+    sceneProcessingConfigSystemComponent.Activate();
+
+    const AZStd::vector<AZ::SceneProcessingConfig::SoftNameSetting*>* result = nullptr;
+    AZ::SceneProcessingConfig::SceneProcessingConfigRequestBus::BroadcastResult(
+        result,
+        &AZ::SceneProcessingConfig::SceneProcessingConfigRequests::GetSoftNames);
+    EXPECT_EQ(result->size(), 2);
+    EXPECT_EQ(result->at(0)->GetVirtualType(), "LODMesh1");
+    EXPECT_EQ(result->at(1)->GetVirtualType(), "Ignore");
+
+    sceneProcessingConfigSystemComponent.Deactivate();
+
+    RemoveReflectedTypes(registrationContext.get(), serializeContext.get());
+}
+
+TEST_F(SceneProcessingConfigTest, SceneProcessingConfigSystemComponent_SoftNameSettings_IgnoreDuplicateSoftNameSettings)
+{
+    const char* settings = R"JSON(
+    {
+        "O3DE": {
+            "AssetProcessor": {
+                "SceneBuilder": {
+                    "NodeSoftNameSettings": [
+                        {
+                            "pattern": {
+                                "pattern": "^.*_[Ll][Oo][Dd]_?1(_optimized)?$",
+                                "matcher": 2
+                            },
+                            "virtualType": "LODMesh1",
+                            "includeChildren": true
+                        },
+                        {
+                            "pattern": {
+                                "pattern": "^.*_[Ll][Oo][Dd]_?1(_optimized)?$",
+                                "matcher": 2
+                            },
+                            "virtualType": "LODMesh1",
+                            "includeChildren": true
+                        },
+                        {
+                            "pattern": {
+                                "pattern": "^.*_[Ll][Oo][Dd]_?2(_optimized)?$",
+                                "matcher": 2
+                            },
+                            "virtualType": "LODMesh2",
+                            "includeChildren": true
+                        }
+                    ]
+                }
+            }
+        }
+    }  
+    )JSON";
+    m_settingsRegistry->MergeSettings(settings, AZ::SettingsRegistryInterface::Format::JsonMergePatch);
+
+    AZStd::unique_ptr<AZ::SerializeContext> serializeContext = AZStd::make_unique<AZ::SerializeContext>();
+    AZStd::unique_ptr<AZ::JsonRegistrationContext> registrationContext = AZStd::make_unique<AZ::JsonRegistrationContext>();
+    m_settingsRegistry->SetContext(serializeContext.get());
+    m_settingsRegistry->SetContext(registrationContext.get());
+    ReflectTypes(registrationContext.get(), serializeContext.get());
+
+    AZ_TEST_START_TRACE_SUPPRESSION;
+    AZ::SceneProcessingConfig::SceneProcessingConfigSystemComponent sceneProcessingConfigSystemComponent;
+    AZ_TEST_STOP_TRACE_SUPPRESSION(1);
```

Comment:
Can you add a comment for what this trace suppression is suppressing? It helps a lot with maintenance if the test starts failing due to more or less trace messages being suppressed.